### DISCUSSION TITLE
RV Constant Distribution Bug

### DISCUSSION THREAD

**Discussion by onigon999** [NHERI-SimCenter/SimCenterCommon#301] [EE-UQ]
I'm having a problem on RV tab, where a random variable won't generate a number when distribution is set to constant, thus giving results of 0. 
![constant](https://github.com/user-attachments/assets/89d871ae-fb71-4779-a6d3-7c01a6331cb2)
![normal](https://github.com/user-attachments/assets/e35fb79e-ba0b-45ca-a387-9ecc2711f4f6)

**Comment by fmckenna** [comment]
thanks for pointing out the issue. For now just remove that random variable and let the value come in from the script. frank

**Comment by yisangriB** [comment]
Hi, thanks again for reporting the issue.

After investigating, we believe this bug has been resolved in the latest version of [EE-UQ (v4.1.0)](https://www.designsafe-ci.org/data/browser/tapis/designsafe.storage.community/%2FSimCenter%2FSoftware%2FEE_UQ), which was released on 10/9. We were able to reproduce the reported bug only in the previous version (v.3.6.0).

Please let us know if the error persists in the latest version of EE-UQ.

Sang-ri

**Reply by onigon999** [reply]
Hi, thanks for responding.

I am currently using V4.1.0 when I found the bug.

**Reply by yisangriB** [reply]
Thank you for the information. To help us reproduce the results, can you please share the screenshots of the UQ, EVT, and SIM tabs?

Additionally, before proceeding, could you please confirm whether the modified date of EE-UQ.exe appears as October 8th (or a date close to it, depending on the time zone) in Windows file explorer? This is to ensure that we are looking at the latest build.

<img width="464" alt="EE-UQ_date" src="https://github.com/user-attachments/assets/939ecaf0-cc1f-46a6-a417-517402587993">


Apologies for the inconvenience!
